src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo
+}
```

Comment:
не забывай точку с запятой
```suggestion
interface Props {
  todo: Todo;
}
```